Userland/Libraries/LibWeb/WebDriver/TimeoutsConfiguration.cpp

```diff
@@ -48,42 +48,42 @@ ErrorOr<TimeoutsConfiguration, Error> json_deserialize_as_a_timeouts_configurati
     // 3. If value has a property with the key "script":
     if (value.as_object().has("script"sv)) {
         // 1. Let script duration be the value of property "script".
-        auto const& script_duration = value.as_object().get_deprecated("script"sv);
+        auto const& script_duration = value.as_object().get("script"sv);
 
         // 2. If script duration is a number and less than 0 or greater than maximum safe integer, or it is not null, return error with error code invalid argument.
-        if (script_duration.is_number() && (script_duration.to_i64() < 0 || script_duration.to_i64() > max_safe_integer))
+        if (script_duration.has_value() && script_duration->is_number() && (script_duration->to_i64() < 0 || script_duration->to_i64() > max_safe_integer))
             return Error::from_code(ErrorCode::InvalidArgument, "Invalid script duration");
-        if (!script_duration.is_number() && !script_duration.is_null())
+        if (script_duration.has_value() && !script_duration->is_number() && !script_duration->is_null())
             return Error::from_code(ErrorCode::InvalidArgument, "Invalid script duration");
 
         // 3. Set timeouts’s script timeout to script duration.
-        timeouts.script_timeout = script_duration.is_null() ? Optional<u64> {} : script_duration.to_u64();
+        timeouts.script_timeout = (!script_duration.has_value() || script_duration->is_null()) ? Optional<u64> {} : script_duration->to_u64();
     }
 
     // 4. If value has a property with the key "pageLoad":
     if (value.as_object().has("pageLoad"sv)) {
         // 1. Let page load duration be the value of property "pageLoad".
-        auto const& page_load_duration = value.as_object().get_deprecated("pageLoad"sv);
+        auto page_load_duration = value.as_object().get("pageLoad"sv);
```

Comment:
This (and `implicit_duration` below) could be:
```c++
auto page_load_duration = value.as_object().get_integer<i64>("pageLoad"sv);
```

Then the check below becomes:
```c++
if (!page_load_duration.has_value() || *page_load_duration < 0 || *page_load_duration > max_safe_integer)
```

And then we can safely cast to `u64` for storage:
```c++
timeouts.page_load_timeout = static_cast<u64>(*page_load_duration);